helm/hpcc/templates/dali.yaml

```diff
@@ -28,6 +28,7 @@ spec:
               ]
 {{ include "hpcc.addSentinelProbes" . | indent 8 }}
 {{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 8 }}
+{{ include "hpcc.addResources" (dict "me" .resources) | indent 8 }}
```

Comment:
should be {{- I think. Otherwise it's always going to generate a newline here, i.e. even if no resources.